### ISSUE TITLE
Unable to double-click to zoom on first page

### ISSUE THREAD

**Issue by mirkancal** [alhappler/alh_pdf_view#39]
When you open a pdf and you double tap, it doesn't zoom. You need to swipe to second page and come back to make double tap zoom work.

**Comment by andreBalh** [comment]
Hi @mirkancal 
Thank you for opening the issue. 
Could you provide more information about the device where that issue happened? @mirkancal

**Comment by andreBalh** [comment]
I checked your issue. It looks like the double tap is just zooming out a little bit. That means that the default zoom value is wrong and has to be fixed.
I will try to fix that. @mirkancal

**Comment by mirkancal** [comment]
@andreBalh thank you. Interesting, on my end double tap doesn't do anything on initial page but pinch to zoom works immediately. To make double tap work, I need to move to another page.

**Comment by andreBalh** [comment]
I think I could fix the issue, could you change the following for `alh_pdf_view` in your `pubspec.yaml` and try that update if that fix your issue? @mirkancal 

```yaml
... 
alh_pdf_view: 
  git: 
    url: https://github.com/alhappler/alh_pdf_view.git 
    ref: dcf69a413d7f27ccf21adc7ffe80f7a48b3ed76e 
...
```

**Comment by mirkancal** [comment]
Thank you, seems fixed to me.

**Comment by andreBalh** [comment]
Should be fixed with version `1.2.1`. @mirkancal
